x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -34,3 +34,31 @@ export const manifestType: SavedObjectsType = {
   mappings: manifestSavedObjectMappings,
   migrations,
 };
+
+export const unifiedManifestSavedObjectType = ManifestConstants.UNIFIED_SAVED_OBJECT_TYPE;
+
+export const unifiedManifestSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
+  properties: {
+    artifactIds: {
+      type: 'text',
+    },
+    policyId: {
+      type: 'keyword',
+    },
+    semanticVersion: {
+      type: 'text',
+    },
+    created: {
+      type: 'date',
+    },
+  },
+};
```

Comment:
Q: do we need a migration like the old version?

Reply:
Not until we commit this new SO and make some changes to the structure.